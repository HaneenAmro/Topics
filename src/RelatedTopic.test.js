
import renderer from 'react-test-renderer';
import RelatedTopics from './RelatedTopic.js';


describe('My compnent', () => {
    const testProps = [
        {
            "name": "react",
            "stargazerCount": 51339
        },
        {
            "name": "vue",
            "stargazerCount": 37685
        },
    ];

    it('it works', () => {
        const tree = renderer.create(<RelatedTopics relName = {testProps[0].name} relCount = {testProps[0].stargazerCount} />).toJSON();
        expect(tree).toMatchSnapshot();
    });
});
