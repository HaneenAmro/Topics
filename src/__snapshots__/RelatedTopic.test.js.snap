// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`My compnent it works 1`] = `
<div
  className="sc-bdnxRM eXgWxp"
>
  <div
    className="MuiPaper-root MuiAccordion-root MuiAccordion-rounded MuiPaper-elevation1 MuiPaper-rounded"
  >
    <div
      aria-disabled={false}
      aria-expanded={false}
      className="MuiButtonBase-root MuiAccordionSummary-root"
      onBlur={[Function]}
      onClick={[Function]}
      onDragLeave={[Function]}
      onFocus={[Function]}
      onKeyDown={[Function]}
      onKeyUp={[Function]}
      onMouseDown={[Function]}
      onMouseLeave={[Function]}
      onMouseUp={[Function]}
      onTouchEnd={[Function]}
      onTouchMove={[Function]}
      onTouchStart={[Function]}
      role="button"
      tabIndex={0}
    >
      <div
        className="MuiAccordionSummary-content"
      >
        <p
          className="MuiTypography-root MuiTypography-body1"
        >
          <span
            className="sc-gtsrHT gxLBAN"
          >
            <strong>
              Topic Name:
            </strong>
            react
          </span>
        </p>
      </div>
    </div>
    <div
      className="MuiCollapse-container MuiCollapse-hidden"
      style={
        Object {
          "minHeight": "0px",
        }
      }
    >
      <div
        className="MuiCollapse-wrapper"
      >
        <div
          className="MuiCollapse-wrapperInner"
        >
          <div
            role="region"
          >
            <div
              className="MuiAccordionDetails-root"
            >
              <p
                className="MuiTypography-root MuiTypography-body1"
              >
                <span
                  className="sc-gtsrHT gxLBAN"
                >
                  <strong>
                    Stargazer count:
                  </strong>
                  51339
                </span>
              </p>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
